Umbraco-Cloud/release-notes/2022-01-06-ReleaseNote.md

```diff
@@ -0,0 +1,28 @@
+# January 6, 2022  Release Notes
+
+_Break project reference + See project information + Various improvements_
+
+### Key Takeaways
+- **Break reference between baseline and child project** - As an admin you can now break the reference between a baseline and child projects
+- **New Project Overview Page** - See the project information such as plan, payment status for your project on the new project overview page.
+- **Various tweaks and improvements** - We have since December provided a lot of small changes and improvements.
+
+### Break reference between baseline and child project
+Umbraco Cloud Portal offers a powerful baseline-child relationship between projects. It is very similar to a fork (forked repository) on GitHub in that we create a clone of an existing project while maintaining a connection between the two projects.
+
+If you at some point **want to break this connection** between the baseline and one of its child projects it is now really easy for an user with the role of admin to do so. But please be aware that the **action can not be undone**.
+
+![Break reference between baseline and child project (1)](https://user-images.githubusercontent.com/93588665/149168277-e7ffb2e1-34c3-411f-9962-e834150f62d1.gif)
+
+### New Project Overview Page
```

Comment:
Could you also link headline to this page: https://our.umbraco.com/documentation/Umbraco-Cloud/Getting-Started/#project-overview